Sources/TuistDependencies/Carthage/CarthageInteractor.swift

```diff
@@ -236,18 +236,56 @@ public final class CarthageInteractor: CarthageInteracting {
             from: pathsProvider.temporaryCarthageBuildDirectory,
             to: pathsProvider.destinationCarthageDirectory
         )
+
+        if dependencies.options.contains(.useXCFrameworks) {
+            // save build xcFrameworks to copyPath directories
+            try copyXCFrameworksToPaths(
+                from: pathsProvider.destinationCarthageDirectory,
+                dependencies: dependencies,
+                dependenciesDirectory: dependenciesDirectory
+            )
+        }
```

Comment:
Could you provide more context on this change? What this has to do with the excluding resources stuff?